src/robot/model/testsuite.py

```diff
@@ -27,6 +27,10 @@
 from .tagsetter import TagSetter
 from .testcase import TestCase, TestCases
 
+from typing import Any, TYPE_CHECKING, Dict, List, Union
+if TYPE_CHECKING:
+    from robot.model.visitor import SuiteVisitor
+
```

Comment:
- Standard module imports should be before internal imports.
- We discussed on #devel about using `'x | y'` `'list[x]'` style instead of  `Union[x, y]` and `list[x]`. That would avoid lot of `from typing` imports, although in this case we anyway needed to import `TYPE_CHECKING`.

Reply:
Should I switch to stringifying all the type hints in the file?